radicle-httpd/src/api.rs

```diff
@@ -148,6 +174,23 @@ mod project {
     use radicle::identity::Id;
     use radicle::prelude::Did;
 
+    /// Detailed information on issue states
+    #[derive(Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct IssueState {
+        pub open: usize,
+        pub closed: usize,
+    }
```

Comment:
I'd suggest creating methods to construct these from the `Issues` and `Patches` types.

Reply:
Done here ca3c21b967e60f644321f4061d8f9ab5ce95bff7